Wrappers/Python/cil/plugins/astra/operators/AstraProjector2D.py

```diff
@@ -37,30 +30,27 @@ def __init__(self, image_geometry, acquisition_geometry, device):
         :param device: The device to run on 'gpu' or 'cpu'
```

Comment:
Are we going to change it to the new doc style? If not then we need to create an issue for these files.

Reply:
I believe we established that we would change as we go as opposed to changing past code.

Reply:
I suppose the answer is yes, we will update it, but not in this PR, because I believe we need to merge CIL-ASTRA in CIL ASAP.